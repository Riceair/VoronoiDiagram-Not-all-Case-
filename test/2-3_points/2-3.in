#雙點測試 垂直
2
200 200
200 400