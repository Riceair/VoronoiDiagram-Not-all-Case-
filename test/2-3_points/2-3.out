P 200 200
P 200 400
E -125800 300 126200 300
